src/queries/usePendingCursorOperation.tsx

```diff
@@ -80,10 +80,9 @@ export function usePendingCursorOperation<T = Document>(
   const [liveQueryEvent, setLiveQueryEvent] = useState<
     LiveQueryEvent | undefined
   >();
-  const [liveQuery, setLiveQuery] = useState<LiveQuery | undefined>();
+  const liveQueryRef = useRef<LiveQuery>(undefined);
```

Comment:
Undefined is the default value, so there's no need to explicitly define it as an input param.
```suggestion
  const liveQueryRef = useRef<LiveQuery>();
```